.travis.yml

```diff
@@ -1,3 +1,10 @@
+if: tag IS present OR type = pull_request OR (branch = master AND type = push)   # we only CI the master, tags and PRs
+
+stages:
+- test
+- name: deploy
+  if: repo = tox-dev/tox AND branch = master AND tag IS present
```

Comment:
Only deploy when tagging a commit on the master branch.

Reply:
Interesting!

About tagging commit on master: I think it would be better to tag the commit at the HEAD on the release PR. This way you guarantee that you are publishing exactly what was tested on the PR.

Reply:
https://github.com/pytest-dev/pytest/pull/3666 😉 Thanks!